test/run_as_user/resources/root.json

```diff
@@ -0,0 +1,18 @@
+{
+  "agent": {
+    "run_as_user": "root",
+    "logfile": ""
+  },
+  "metrics": {
+    "metrics_collected": {
+      "disk": {
+        "measurement": [
+          "used_percent"
+        ],
+        "resources": [
```

Comment:
I will avoid using `resources=*` with disk. Better to reduces the amount of metrics we send to CW (e.g Mem Metrics, etc) to validate the test

Reply:
We only run for 15 seconds instead of the flush time of 60 seconds thus cw should not get any metrics. We are only testing process is the correct user.